fp/battle_modifier.py

```diff
@@ -1215,6 +1221,13 @@ def start_volatile_status(battle, split_msg):
         pkmn.volatile_statuses.append(volatile_status)
 
     if volatile_status == constants.SUBSTITUTE:
+        if split_msg[4] == "[from] move: Shed Tail":
```

Comment:
```suggestion
        if len(split_msg) >= 5 and split_msg[4] == "[from] move: Shed Tail":
```

Because of the way I've written this, all checks like this need to verify the length of the incoming messages. This probably fails some existing tests